src/styles/blocks/review.css

```diff
@@ -0,0 +1,16 @@
+.review {
+  align-self: flex-start;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  height: 16px;
+}
+
+.review__num {
```

Comment:
also this element has default margins and they are falling out of the container. not good. maybe use a mix and set margin for the entire review section (.produc__review)

Reply:
Done